WebRoot/js/ajax/dwt/widgets/DwtControl.js

```diff
@@ -2769,26 +2769,29 @@ function() {
  */
 DwtControl.prototype._position =
 function(loc) {
-	this._checkState();
-	var sizeShell = this.shell.getSize();
-	var sizeThis = this.getSize();
-	var x, y;
-	if (!loc) {
-		// if no location, go for the middle
-		x = Math.round((sizeShell.x - sizeThis.x) / 2);
-		y = Math.round((sizeShell.y - sizeThis.y) / 2);
-	} else {
-		x = loc.x;
-		y = loc.y;
-	}
-	// try to stay within shell boundaries
-	if ((x + sizeThis.x) > sizeShell.x) {
-		x = sizeShell.x - sizeThis.x;
-	}
-	if ((y + sizeThis.y) > sizeShell.y) {
-		y = sizeShell.y - sizeThis.y;
-	}
-	this.setLocation(x, y);
+      this._checkState();
+      var sizeShell = this.shell.getSize();
+      var sizeThis = this.getSize();
+      var x, y;
+      if(sizeThis)
+      {
```

Comment:
@barrydegraaff , can you please make opening of `{` on the same line as `if`, like `if {`. 

This is to make sure, we all are on same coding convention. 

Thanks